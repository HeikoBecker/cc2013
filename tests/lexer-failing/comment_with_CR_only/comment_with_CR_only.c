// CR nowcomment_with_CR_only int this_must_show_up;
/* block comment with CR *////**/
