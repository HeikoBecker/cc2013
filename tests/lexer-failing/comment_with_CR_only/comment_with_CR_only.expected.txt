comment_with_CR_only.c:2:1: keyword int
comment_with_CR_only.c:2:5: identifier this_must_show_up
comment_with_CR_only.c:2:22: punctuator ;
